src/static_api.rs

```diff
@@ -48,6 +48,9 @@ impl<'a> Generator<'a> {
             members.sort_by_key(|member| member.github.to_lowercase());
             members.sort_by_key(|member| !member.is_lead);
 
+            let mut github_teams = team.github_teams();
+            github_teams.sort();
```

Comment:
What do we gain by this? Determinism?

Reply:
Yeah, all the things outputted by the API are currently deterministic. Other than generally being nicer, the synchronization code on RCS will run only if the API content changed, and having it deterministic avoids unnecessary runs.